README.md

```diff
@@ -36,31 +36,11 @@
 </p>
 </br>
 
-Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards. [Get started →.](#getting-started)
+Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards. Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. [Get started →.](#getting-started)
 
-Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**.
+<img src="images/semgrep-main-image.jpg" alt="Semgrep platform image"/>
```

Comment:
This image and coloration is quite lovely. wydt of us leading with a terminal output image since this is the home of our OSS project? Save this photo as a value-pitch in the ui section?

Reply:
Thanks. Our terminal output isn't good just yet, but once we have it, we can certainly come back to this.